packages/atlas/src/views/viewer/AuctionView/AuctionView.tsx

```diff
@@ -0,0 +1,339 @@
+import { DrawerHeader } from '@/components/DrawerHeader'
+import { Information } from '@/components/Information'
+import { NftCard } from '@/components/NftCard'
+import { Pill } from '@/components/Pill'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { SvgAlertsWarning24 } from '@/components/_icons'
+import { JoyTokenIcon } from '@/components/_icons/JoyTokenIcon'
+import { SvgJoystreamLogoShort } from '@/components/_illustrations'
+import { TextField } from '@/components/_inputs/TextField'
+import { SkeletonLoader } from '@/components/_loaders/SkeletonLoader'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { useMsTimestamp } from '@/hooks/useMsTimestamp'
+import { useAuction } from '@/providers/auction/hooks'
+import { useJoystream, useTokenPrice } from '@/providers/joystream'
+import { useUser } from '@/providers/user'
+import { cVar } from '@/styles'
+import { formatDurationShort } from '@/utils/time'
+
+import {
+  ActionBarCell,
+  ActiveBidWrapper,
+  BidderName,
+  Container,
+  Content,
+  CurrentBidAvatar,
+  CurrentBidJoyToken,
+  CurrentBidWrapper,
+  Divider,
+  EndingTime,
+  FlexWrapper,
+  Header,
+  InnerContainer,
+  Messages,
+  MinimumBid,
+  MinimumBidWrapper,
+  NftPreview,
+  PaymentSplitValues,
+  PaymentSplitWrapper,
+  PlaceBidWrapper,
+  Row,
+  StyledActionBar,
+  Timer,
+} from './AuctionView.styles'
+
+const DUMMY_NFT_TILE_PROPS = {
+  role: 'owner' as const,
+  auction: 'none' as const,
+  thumbnail: { thumbnailUrl: 'https://placedog.net/360/203' },
+  creator: { assetUrl: 'https://placedog.net/100/100?random=1', name: 'Jane' },
+  owner: { assetUrl: 'https://placedog.net/100/100?random=2', name: 'Kate' },
+  loading: false,
+  fullWidth: false,
+}
+
+const TRANSACTION_FEE = 19
+const FIXED_PRICE = 500
+const MINIMUM_BID = 301
+const END_TIME = Date.now() + 80000
+const BID = {
+  bidder: {
+    name: 'Mike Shipa',
+  },
+  amount: 341,
+}
+
+export const AuctionView: React.FC = () => {
+  const [type, setType] = useState<'auction' | 'buy_now'>('auction')
+  const { isAuctionOpen, setAuctionOpen } = useAuction()
+  const mdMatch = useMediaMatch('md')
+  const { convertToUSD } = useTokenPrice()
+  const { joystream } = useJoystream()
+  const { activeMembership } = useUser()
+  const [placedBid, setPlacedBid] = useState<number | undefined>()
+  const [accountBalance, setAccountBalance] = useState<number | undefined>()
+  const timestamp = useMsTimestamp()
+  const timeLeft = Math.trunc((END_TIME - timestamp) / 1000)
+
+  const onPlaceBid = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setPlacedBid(Number(event.target.value))
+  }
+
+  useEffect(() => {
+    const init = async () => {
+      if (activeMembership) {
+        const balance = await joystream?.getAccountBalance(activeMembership?.controllerAccount)
+        setAccountBalance(balance)
+      }
+    }
+    init()
+  }, [activeMembership, joystream])
```

Comment:
I think it may be safer to use a subscription here? Currently we will fetch this only on first render, but the lifetime of this component may be long since we render it at all times, just not show it